catmux/session.py

```diff
@@ -101,11 +101,30 @@ def _parse_common(self):
         if "common" in self.__yaml_data and self.__yaml_data["common"]:
             common = self.__yaml_data["common"]
             if "before_commands" in common and common["before_commands"]:
-                self._before_commands = common["before_commands"]
+                # self._before_commands = common["before_commands"]
+                self._parse_before_commands(common["before_commands"])
```

Comment:
No need to keep old code around.
```suggestion
                self._parse_before_commands(common["before_commands"])
```